src/Employer/Employer.Web/Configuration/ConfigurationExtensions.cs

```diff
@@ -91,7 +89,10 @@ public static void AddMvcService(this IServiceCollection services, IHostingEnvir
 
                 opts.AddTrimModelBinderProvider(loggerFactory);
             })
-            .AddFluentValidation(fv => fv.RegisterValidatorsFromAssemblyContaining<Startup>())
+                .AddFluentValidation(fv =>
+                {                    
+                    fv.RegisterValidatorsFromAssemblyContaining<Startup>();                    
```

Comment:
Do we need this line now that you are registering the dependency more explicitly?

Reply:
It was existing code, so dint want to change it!
Shall I remove it?

Reply:
Removed.